README.md

```diff
@@ -75,6 +75,73 @@ The PMEM-CSI driver supports running in different modes, which can be controlled
 
 * **_Unified_** mode is intended to run the driver in a single host, mostly for testing the driver in a non-clustered environment.
 
+### Volume Persistency
```

Comment:
The intend behind this section is to describe the problem without using Kubernetes-specific terminology.

Only when it comes to comments about the actual implementation does it refer to Kubernetes.